Sparkbox Sprint 8

kaseybon opened the pull request:
# READY FOR REVIEW

## Summary
- WYSIWYG Styles [STN-127](https://sparkbox.atlassian.net/browse/STN-127) | PR #494 
- Local Footer Styles and Implementation [STN-172](https://sparkbox.atlassian.net/browse/STN-172) | PR #496 
- QA Feedback from last sprint [STN-184](https://sparkbox.atlassian.net/browse/STN-184) | PR #497 
- Hero option 1 styles [STN-125](https://sparkbox.atlassian.net/browse/STN-125) | PR #495 

✅ Passing CodeClimate: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/498

## Need Review By (Date)
asap (2/25)

## Urgency
high

## Steps to Test
- ⚠️ TODO: We've verified that this branch passes the code climate and circle CI tests against the develop branch.

1. See other PRs "steps to test" if you'd like to test locally for those changes.
2. Verify code looking at the file changes here.
3. Go into `humsci_basic` folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

Commit:
STN-127 : WYSIWYG Text Area Styles (#494)

* feat: add WYSIWYG styles

JIRA STN-127 https://sparkbox.atlassian.net/browse/STN-127

* feat: add config settings so that the WYSIWYG displays the correct theme styles within the editor

* refactor: move styles out of utilities and into elements

The WYSIWYG styles are more of an element such as a form or table, than a utility.

* feat: move blockquote styles outside of the field-hs-text-area class to allow the styles to render within the WYSIWYG editor

kaseybon committed:
STN-172: Local Footer (#496)

* feat: local footer markup and styles

- local footer includes a default light style and a dark variant
- local footer blocks will use the decanter flexbox grid
- grid classes can be added when configuring blocks
- move font styles from paragraph tag into the body element
- refactor button class to work with anchor tags
- style font-awesome elements

STN-172

* fix: z-index issue with lockup

- the lockup was overlaying the Drupal admin bar at small-medium screen sizes

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

Commit:
feat(STN-184): qa feedback styling changes (#497)

- refine secondary nav icons
- add pointers to button filters hover
- refactor and refine 3 col w image layout max width and alignment
- stylelint fixes
- ensure body/description/subtitle font size are the same for vertical/horizontal/date/structured card
- delete blue category bar styling on people structured card (education area)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

Commit:
feat(STN-125):  hero 1 styles (#495)

* feat(STN-125): hero option 1 pattern styles

- implements css grid hero option 1 styles with pseudo element color blocks
- adds border button mixin and styling for buttons and links
- alters some of the pattern UI html twig markup for the option 1 hero

* fix(STN-125): run postcss on compiled CSS theme files

* feat(STN-125): adds our supported browsers to .browserslist config

- if you run `npx browserslist` you get out the supported browsers that we will autoprefix for. This is more than we stated we support, but it doesnt hurt to support the top 1% of modern browsers either:

and_chr 78
and_ff 68
and_qq 1.2
and_uc 12.12
android 76
baidu 7.12
chrome 78
chrome 77
chrome 76
chrome 49
edge 18
edge 17
firefox 70
firefox 69
firefox 68
ie 11
ie_mob 11
ios_saf 13.2
ios_saf 13.0-13.1
ios_saf 12.2-12.4
kaios 2.5
op_mini all
op_mob 46
opera 64
opera 63
safari 13
safari 12.1
safari 5.1
samsung 10.1
samsung 9.2

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

kaseybon committed:
chore: commit compiled css

kaseybon committed:
fix: resolve codeclimate errors

pookmish commented on docroot/themes/humsci/humsci_colorful/theme-settings.php:
```diff
@@ -0,0 +1,37 @@
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\Core\Link;
+use Drupal\Core\Render\Markup;
```
unsused `use` statements

kaseybon replied:
@pookmish I pushed a fix removing those statements.

pookmish reviewed (changes requested):
I've also noticed the behat tests are failing. I'll open a PR and request that be fixed as part of this

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647
